src/Laravel/NorthstarUserProvider.php

```diff
@@ -67,8 +62,7 @@ public function retrieveByCredentials(array $credentials)
      */
     public function validateCredentials(Authenticatable $user, array $credentials)
     {
-        $token = $this->northstar->authorizeByPasswordGrant($credentials);
-
-        return ! is_null($token);
+        // Is this weird? Yes.
+        return true;
```

Comment:
Ah gotcha. That's why you mentioned the `NorthstarAuth` class. coolio :) Thanks for clarifying!